packages/cli/src/helpParamGenerator.ts

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+"use strict";
+
+import { Result, FxError, err, ok, Inputs, Tools, Stage, Platform, QTreeNode, Question, isAutoSkipSelect, SingleSelectQuestion, MultiSelectQuestion, OptionItem } from "@microsoft/teamsfx-api";
+
+import { FxCore } from "@microsoft/teamsfx-core";
+import AzureAccountManager from "./commonlib/azureLogin";
+import AppStudioTokenProvider from "./commonlib/appStudioLogin";
+import GraphTokenProvider from "./commonlib/graphLogin";
+import CLILogProvider from "./commonlib/log";
+import DialogManagerInstance from "./userInterface";
+import { CliTelemetry } from "./telemetry/cliTelemetry";
+import CLIUIInstance from "./userInteraction";
+import { flattenNodes, getSingleOptionString, toYargsOptions } from "./utils";
+import { Options } from "yargs";
+
+const rootFolderNode = new QTreeNode({
+  type: "folder",
+  name: "folder",
+  title: "Select root folder of the project",
+  default: "./",
+});
+
+export class HelpParamGenerator {
+  private static core: FxCore | undefined;
+  private static questionsMap: Map<string, QTreeNode> = new Map<string, QTreeNode>();
+  public static async activate(): Promise<Result<FxCore, FxError>> {
```

Comment:
Can it be changed to sync function?

Reply:
seems it can. thx for the reminder, will commit that in the merged branch.